README.md

```diff
@@ -1 +1,24 @@
-# hw01
\ No newline at end of file
+# About Me
+Hello, my name is Tianyi Qian.  
+
+### Educational Background  
```

Comment:
Merit: I really like the way you use subheadings with different font to distinguish different topics/headings. This has made your biography clear and easy for readers to see the structure of it.